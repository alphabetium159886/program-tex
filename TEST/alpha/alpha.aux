\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {section}{\numberline {1}对函数$f(z) = 1/z$做runge近似}{1}{section.1}\protected@file@percent }
\newlabel{f2}{{1}{2}{对函数$f(z) = 1/z$做runge近似}{equation.1.1}{}}
\newlabel{f2}{{2}{2}{对函数$f(z) = 1/z$做runge近似}{equation.1.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2}$f(z) = \frac  {1}{(z+1)^2}$做runge近似}{2}{section.2}\protected@file@percent }
\bibstyle{IEEEtran}
\bibdata{lecktion0}
\gdef \@abspage@last{4}
