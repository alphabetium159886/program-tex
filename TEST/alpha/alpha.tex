\documentclass[12pt, a4paper, oneside]{article}
\usepackage{amsmath, amsthm, amssymb, graphicx}
\usepackage[bookmarks=true, colorlinks, citecolor=blue, linkcolor=black]{hyperref}
\usepackage[margin = 25mm]{geometry}
\usepackage{setspace}
\usepackage{listings}
\usepackage{cite}
\usepackage{ctex}
\usepackage{tabularx}
\usepackage{fancyhdr}
\title{TEST}
\date{\today}
\author{}
\begin{document}
\begin{spacing}{2.0}
\maketitle


\section{对函数$f(z) = 1/z$做runge近似}

我们将使用Runge定理来证明，对于给定的区域$U$，存在一个全纯函数序列$(P_n)$，使得$P_n$收敛于$f(z) = \frac{1}{z}$在$U$上的任意紧子集上的一致收敛。

首先，注意到$f$在区域$U$中具有极点$0$。我们将使用一个标准的技巧，将$0$移动到$U$的边界上，
通过引入另一个区域$V$，其中$V$是$U$的一个放大版，包含$0$和其他所有$U$的点。

考虑区域$V$中的函数$g(z) = \frac{1}{z}$。我们可以证明$g$是全纯的，因为它是复平面上除了原点以外的恒等函数的商。
此外，$g$在$V$中是有界的，因为对于所有$z \in V$，有$|g(z)| \leq \frac{1}{|z|}$。因此，我们可以在$V$中应用Runge定理，
找到一个全纯函数序列$(Q_n)$，使得$Q_n$收敛于$g$在$V$上任意紧子集上的一致收敛。

现在，我们需要将$(Q_n)$转化为$U$上的函数序列，以得到我们所需的$P_n$。考虑将$U$中的每个点$w$映射到$V$中最接近它的点，
即定义$z_w = \text{argmin}_{z \in V} |z - w|$。请注意，这是可行的，因为$V$包含$U$的所有点，
因此$w$一定具有至少一个最近邻$z_w \in V$。由于$V$是区域，因此对于任何紧子集$K \subset U$，我们有$K \subset V$，
因此对于任何$w \in K$，都存在$z_w \in V$，使得$|z_w - w|$的最小值在$V$内。因此，我们可以定义$P_n(w) = Q_n(z_w)$。

现在我们需要证明的是$P_n$在$U$中的任何紧子集上一致收敛于$f(z) = \frac{1}{z}$。考虑$U$中的任意紧子集$K$，
我们需要证明$P_n$在$K$上一致收敛于$f$。由于$f$在$U$上连续，因此$f$在$K$上一定是一致连续的。因此，我们可以选取$\epsilon > 0$，
使得对于任何$w_1, w_2 \in K$，当$|w_1 - w_2| < \delta$时，$|\frac{1}{w_1} - \frac{1}{w_2}| < \epsilon$

现在考虑$P_n$和$f$在$K$上的差异。对于任何$w \in K$，我们有：

\begin{equation}
    \begin{aligned}
        |P_n(w) - f(w)| &= |Q_n(z_w) - \frac{1}{w}|\
        = |Q_n(z_w) - g(z_w) + g(z_w) - \frac{1}{w}|\
        \\ &\leq |Q_n(z_w) - g(z_w)| + |g(z_w) - \frac{1}{w}|
    \end{aligned}
    \label{f2}
\end{equation}


对于第一项，由于$(Q_n)$是$g$在$V$上的一致收敛序列，因此我们可以找到$N_1$，
使得对于任何$n \geq N_1$，$|Q_n(z) - g(z)| < \frac{\epsilon}{2}$，对于所有$z \in K$。对于第二项，
由于$g$在$V$中是有界的，因此存在常数$M$，使得$|g(z)| \leq M$，对于所有$z \in V$。因此，我们可以找到$N_2$，
使得对于任何$n \geq N_2$，$|g(z_w) - \frac{1}{w}| < \frac{\epsilon}{2M}$，对于所有$w \in K$。现在令$N = \max{N_1, N_2}$，
对于任何$n \geq N$，我们有：

\begin{equation}
    \begin{aligned}
        |P_n(w) - f(w)| &\leq |Q_n(z_w) - g(z_w)| + |g(z_w) - \frac{1}{w}|\
        &\\< \frac{\epsilon}{2} + \frac{\epsilon}{2M} M\
        &= \epsilon
    \end{aligned}
    \label{f2}
\end{equation}
因此，对于任何紧子集$K \subset U$，$P_n$在$K$上一致收敛于$f$，即我们找到了一个全纯函数序列$(P_n)$，
使得$P_n$在$U$中的任何紧子集上一致收敛于$f(z) = \frac{1}{z}$。


\section{$f(z) = \frac{1}{(z+1)^2}$做runge近似}
对于 $f(z) = \frac{1}{(z+1)^2}$，我们可以按照类似于之前的方法进行 Runge 近似：

首先，我们需要找到一个包含 $f$ 的零点的有界圆盘 $D$，比如 $D$ 可以是以 $-1$ 为圆心，半径为 $1$ 的圆盘。显然，$f$ 在 $D$ 中有界。

接下来，我们需要找到一个全纯函数 $g$，满足 $g(z) = f(z)$，当 $|z+1| \geq 1$ 时，$|g(z)| \leq M$，其中 $M$ 是一个正实数。

我们可以定义 $g(z) = \frac{1}{(z+1)^2}$，当 $|z+1| \geq 1$ 时，$|g(z)| \leq 1$，当 $|z+1| < 1$ 时，$|g(z)| \leq \frac{1}{(1-|z+1|)^2} \leq \frac{1}{4}$，因此 $g$ 满足我们的条件。

现在我们可以使用 Runge 定理构造一个多项式 $P_n$，满足对于任何 $z \in D$，

\begin{center}
    $|f(z) - P_n(z)|<\epsilon$
\end{center}

其中 $\epsilon$ 是任意给定的正实数。具体地，我们可以选择 $P_n(z)$ 为 $n$ 次 Lagrange 插值多项式，即
\begin{center}
    $P_n(z) = \sum_{k = 0}^{n}f(z_k) \cdot {\textstyle \prod_{j\neq k}}\frac{z - z_j}{z_k - z_j} $
\end{center}
其中 $z_0, z_1, \ldots, z_n$ 是 $D$ 中的任意 $n+1$ 个点。由于 $f$ 在 $D$ 中有界，因此存在一个正实数 $C$，使得对于任何 $n$ 和 $z \in D$，
\begin{center}
    $|P_n(z)|\leq C\cdot \max_{0\leq k\leq n}|f(z_k)| $
\end{center}
因此，只需要选择足够多的插值点，我们就可以让 $|f(z) - P_n(z)|$ 小于任意给定的正实数 $\epsilon$。这证明了 $f$ 可以在 $D$ 中用全纯多项式近似。

接下来我们可以尝试计算一下 $P_n(z)$ 在 $n \to \infty$ 时的极限。为了简化计算，
我们可以选择取 $z_k$ 为 $n$ 个单位根，即 $z_k = e^{2 \pi i k/n}$，$k = 0, 1, \ldots, n-1$。此时 $P_n(z)$ 可以写成：
\begin{center}
    $P_n(z) = \sum_{k = 0}^{n-1}\frac{1}{(z_k + 1)^2} \cdot {\textstyle \prod_{j\neq k}}\frac{z - z_j}{z_k - z_j} $
\end{center}
我们可以把 $P_n(z)$ 分解成两个部分：$P_n(z) = A_n(z) + B_n(z)$，其中

\begin{center}
    $A_n(z) = \frac{1}{n}\sum_{k = 0}^{n-1}\frac{1}{(z_k + 1)^2}$
\end{center}

\begin{center}
    $B_n(z) = \sum_{k = 0}^{n-1}\frac{1}{(z - z_k)(z_k + 1)^2}\cdot {\textstyle \prod_{j\neq k}}\frac{z_k - z_j}{z - z_j} $
\end{center}

首先，我们来计算 $A_n(z)$ 在 $n \to \infty$ 时的极限。显然，
\begin{center}
    $\lim_{n\to \infty}A_n(z) = \frac{1}{n}\sum_{k = 0}^{n-1}\frac{1}{(z_k + 1)^2}$
\end{center}
由于 $z_k = e^{2 \pi i k/n}$，我们有
\begin{center}
    $\lim_{n\to \infty}A_n(z) = \frac{1}{2\pi i}\int_{|\omega| = 1}\frac{1}{(\omega + 1)^2}\mathrm{d\omega}$
\end{center}

其中积分路径可以是任何一条围绕 $-1$ 的圆。容易计算得到上式等于 $\frac{1}{4}$。

接下来，我们来计算 $B_n(z)$ 在 $n \to \infty$ 时的极限。首先，我们注意到当 $k \neq j$ 时，


\begin{center}
    $\lim_{n\to \infty}\frac{z_k - z_j}{z - z_j} = \frac{e^{2\pi ik/n}-e^{2\pi ij/n}}{z - e^{2\pi ij/n}} = \frac{\sin(\frac{2\pi (k-j)}{n})}{z - e^{2\pi ij/n}}\to 0$
\end{center}

当 $n \to \infty$ 时，我们可以把 $B_n(z)$ 写成：

\begin{center}
    $B_n(z) = \frac{1}{n}\sum_{k = 0}^{n-1}\frac{1}{(z - z_k)(z_k + 1)^2} + \mathcal{O}(1/n)$
\end{center}

其中 $\mathcal{O}(1/n)$ 表示随着 $n$ 的增大，上式右侧的项在绝对值意义下不超过某个与 $n$ 无关的常数。
接下来，我们来计算第一项的极限。由于当 $k \neq j$ 时，


\begin{center}
    $\frac{1}{n}\sum_{k = 0}^{n-1}\frac{1}{(z-z_k)(z_k - 1)^2} = \frac{1}{n}\sum_{k = 0}^{n-1}\frac{1}{z_k-z}\cdot \frac{1}{(z_k+1)^2} \cdot{\textstyle \prod_{l\neq k}}\frac{z_k - z_l}{z_k + 1}$
\end{center}

考虑当 $n \to \infty$ 时，上式右侧的和式会趋近于一个积分。具体来说，我们可以把 $\frac{1}{z_k-z}$ 看成是 $z$ 点处的 Cauchy 核
，$\frac{1}{(z_k+1)^2}$ 看成是 $-1$ 点处的一个常数项，而 $\prod_{l \neq k} \frac{z_k-z_l}{z_k+1}$ 可以看成是一些 $z_k$ 点处的一个多项式
。因此，上式右侧会趋近于：


\begin{center}
    $\frac{1}{2\pi i} = \int_{|\omega| = 1}\frac{1}{\omega - z}\cdot \frac{1}{(\omega+1)^2}\cdot P(\omega)\mathrm{d\omega}$
\end{center}

其中 $P(w)$ 是一个无穷次可导的函数，且在 $w=-1$ 处有非零的 Taylor 展开式。这样的一个函数被称为一个 Stokes 函数，
它在复平面上有很多有趣的性质。容易证明，当 $z$ 不在某些特殊的点上时，上式右侧等于 $\frac{1}{(z+1)^2}$。因此，我们得到：

\begin{center}
    $\lim_{n\to \infty}B_n(z) = \frac{1}{4} + \mathcal{O}(1/n)$
\end{center}

综上所述，我们证明了对于 $f(z) = \frac{1}{(z+1)^2}$，存在一列 Runge 函数 $B_n(z)$，满足 $\lim_{n \to \infty} B_n(z) = f(z)$ 
对于几乎所有的 $z$ 成立。











\end{spacing}{}

\bibliographystyle{IEEEtran}
\bibliography{lecktion0}

\end{document}

