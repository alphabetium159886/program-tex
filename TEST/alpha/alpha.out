\BOOKMARK [1][-]{section.1}{\376\377\133\371\121\375\145\160\000f\000\050\000z\000\051\000\040\000=\000\040\0001\000/\000z\120\132\000r\000u\000n\000g\000e\217\321\117\074}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000f\000\050\000z\000\051\000\040\000=\000\040\0001\000\050\000z\000+\0001\000\051\0002\120\132\000r\000u\000n\000g\000e\217\321\117\074}{}% 2
